types/node/index.d.ts

```diff
@@ -2651,6 +2651,7 @@ declare module "dns" {
         family?: number;
         hints?: number;
         all?: boolean;
+        verbatim?: boolean;
```

Comment:
Docs?

Reply:
I'm not sure what you're asking for. If you're referring to the documentation for this option, I linked to it in the original PR comment.